### ISSUE TITLE
Error running automation if it has never been run before

### ISSUE THREAD

**Issue by shadyabhi** [ddaddy/Tesla-Smart-Charging#1]
I'm very new to writing complicated automations, so I'm still investigating on how to fix this correctly, but this is a known issue. 

<img width="1145" alt="image" src="https://github.com/ddaddy/Tesla-Smart-Charging/assets/631797/42793729-953a-4c58-a035-e15c8f159611">

PS: Thank you for creating this repo, I'm using this as a base to make the automation working on my end.

**Comment by ddaddy** [comment]
What do you mean "it's a known issue"?
Is your Home Assistant up to date?

**Comment by shadyabhi** [comment]
@ddaddy Sorry, let me give a better explanation. 

First of all, between two hours ago to now, I've finally configured it and this automation works. Thank you creating this repo, it really helped. 

Everything went super smoothly, but the only change was, I had to run the "[Tesla Smart Charge] Charge on solar power" `once` before it all started working. 

That's the case because if the automation is new, `this.attributes.last_triggered` is set to `Never`, in `HASS's` context, it is set to `None`, that's what the error says. 

<img width="511" alt="image" src="https://github.com/ddaddy/Tesla-Smart-Charging/assets/631797/5afd15aa-e4e1-42e4-9103-f50be78c6802">

So, in this condition check, we need to handle scenario when `this.attributes.last_triggered` is `None`.
